.buildkite/pipeline.yml

```
@@ -6,15 +6,15 @@ agents:
 
 steps:
   - label: ":package: Package Cloudbeat - Snapshot"
-    if: build.branch == 'main' || build.branch =~ /^[0-9]+\.[0-9]+\$/ || build.env("RUN_RELEASE") == "true"
+    if: build.branch == 'main' || build.branch == '8.x' || build.branch =~ /^[0-9]+\.[0-9]+\$/ || build.env("RUN_RELEASE") == "true"
     env:
       WORKFLOW: "snapshot"
     key: "package-snapshot"
     command: "./.buildkite/scripts/package.sh"
     artifact_paths: "build/distributions/*"
 
   - label: ":rocket: Publishing Snapshot DRA artifacts"
-    if: build.branch == 'main' || build.branch =~ /^[0-9]+\.[0-9]+\$/ || build.env("RUN_RELEASE") == "true"
+    if: build.branch == 'main' || build.branch == '8.x' || build.branch =~ /^[0-9]+\.[0-9]+\$/ || build.env("RUN_RELEASE") == "true"
```

nit: We could consider this approach if we don't fear `7.x` and the likes. Please note the `x` in the second char class. 
```suggestion
    if: build.branch == 'main' || build.branch =~ /^[0-9]+\.[0-9x]+\$/ || build.env("RUN_RELEASE") == "true"
```

---

Yeah, I wasn't sure about the generic `number.x` vs. specific `8.x` personally I am fine with both.